src/main/java/com/onarandombox/MultiverseCore/utils/ReflectHelper.java

```diff
@@ -0,0 +1,140 @@
+package com.onarandombox.MultiverseCore.utils;
+
+import org.jetbrains.annotations.Nullable;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+
+/**
+ * Utility class used to help in doing various reflection actions.
+ */
+public class ReflectHelper {
+
+    /**
+     * Try to get the {@link Class} based on its classpath.
+     *
+     * @param classPath The target classpath.
+     * @return A {@link Class} if found, else null.
+     */
+    @Nullable
+    public static Class<?> getClass(String classPath) {
+        try {
+            return Class.forName(classPath);
+        } catch (ClassNotFoundException e) {
+            return null;
+        }
+    }
```

Comment:
Hopefully we're not using this all the time as it is a relatively costly operation. Class.forName is "slow" and should an exception be thrown that will also slow this down quite a bit.

Reply:
We might want to consider a caching strategy here.

Reply:
Yea we do store the result, it should only be used on started, then the results are stored.

Reply:
You can see #2580